applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -300,8 +347,16 @@ check_uploaded_file(Context, _ReqFiles) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec normalize_view_results(wh_json:object(), wh_json:objects()) -> wh_json:objects().
-normalize_view_results(JObj, Acc) ->
-    [wh_json:get_value(<<"value">>, JObj)|Acc].
+normalize_view_results(JObj, Acc) when is_list(Acc) ->
```

Comment:
Spec is now wrong. A 4-tuple is not a JSON key. I'd type it something more like:

```erlang
-type view_result_key() :: {api_binary(), ne_binaries(), api_binary(), ne_binaries()}.  
-type view_result_value() :: wh_json:json_term().                                          
-type view_result() :: {view_result_key(), view_result_value()}.                          
-type view_results() :: [view_result(),...] | [].           

-spec normalize_view_results(wh_json:object(), view_results()) -> view_results().
```